Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'webpacker', '~> 5.0'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'bootsnap', '>= 1.4.4', require: false
 gem 'devise'
+gem 'money-rails', '~>1.12'
```

Comment:
this is unnecessary and a bit of an overkill

Reply:
OK, i get that. Thank You :) I just deleted that gem ;)